xcode/Subconscious/Shared/Library/DummyDataUtilities.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  DummyDataUtilities.swift
+//  Subconscious
+//
+//  Created by Ben Follington on 31/3/2023.
+//
+
+import Foundation
+
+protocol DummyData {
+    static func dummyData() -> Self
+}
```

Comment:
I needed a way to populate all these views, I figure these could be useful to keep growing.